probatus/feature_elimination/feature_elimination.py

```diff
@@ -402,6 +402,7 @@ def fit(
         self,
         X,
         y,
+        groups=None,
         sample_weight=None,
         columns_to_keep=None,
         column_names=None,
```

Comment:
I would add the new param as a last parameter to improve backward compatibility
```suggestion
        sample_weight=None,
        columns_to_keep=None,
        column_names=None,
        groups=None,
```

Reply:
I made the change, thanks! I was following the argument order for [RFECV](https://github.com/scikit-learn/scikit-learn/blob/37ac6788c9504ee409b75e5e24ff7d86c90c2ffb/sklearn/feature_selection/_rfe.py#L635) but you are right it makes more sense to be backward compatible here.